### PR TITLE
Train Pnr Status

### LINKED ISSUE #1123: Live PNR status
# Aim

- By entering the train number and date the script will show the live running status os train. 

# Details

- The live running status will be fetched via API

### PR DESCRIPTION
**Pull request by Amit366**
# Description

Added web scrapping technique to get the PNR status
## Type of change

_Please delete options that are not relevant._

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update
- [ ] Documentation Update

## Checklist:

- [ ] My code follows the style guidelines(Clean Code) of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have created a helpful and easy to understand `README.md`
- [ ] My documentation follows [`Template for README.md`](https://github.com/avinashkranjan/Amazing-Python-Scripts/blob/master/Template%20for%20README.md)
- [ ] I have added the project meta data in the PR template.
- [ ] I have created the ``requirements.txt`` file if needed.

## Project Metadata

`` If there is no-file/nothing to fill the below fields with, then type: none ``

`` Example: `` If no requirements.txt needed/present, then type none in ``Requirments``.

Category:
- [ ] Calculators
- [ ] AI/ML
- [x] Scrappers
- [ ] Social_Media
- [ ] PDF
- [ ] Image_Processing
- [ ] Video_Processing
- [ ] Games
- [ ] Networking
- [ ] OS_Utilities
- [ ] Automation
- [ ] Cryptography
- [ ] Computer_Vision
- [ ] Fun
- [ ] Others

Title: Live PNR Status
Folder: Train_pnr_status

Requirements: selenium, PIL, time, io, cv2

Script: script.py

Arguments : none

Contributor: Amit366

Description: Added web scrapping technique to get the PNR status

### CODE REVIEW

**Comment by Amit366** [comment]
@kaustubhgupta please review

File: Train_pnr_status/Readme.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+### Requirements
+
+    - selenium
+    - PIL
+    - pytesseract
+    - io
+    - numpy
+    - time
+    - cv2
+    - tkinter
```
Comment (kaustubhgupta) [inline Train_pnr_status/Readme.md:43]: put all these requirements in requirements.txt file

File: Train_pnr_status/script.py [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+    lbl = ttk.Label(window, text = "                         ")
+    lbl.grid(column = 1, row = 4) 
+    window.mainloop()
+
+GUI()
+window = tkinter.Tk()
+name = tkinter.StringVar()    
+url="http://www.indianrail.gov.in/enquiry/PNR/PnrEnquiry.html?locale=en"
+
+google = input(r"Enter google executable path");
```
Comment (kaustubhgupta) [inline Train_pnr_status/script.py:42]: which google executable? I am not aware of any google.exe. Can you clarify this?

File: Train_pnr_status/script.py [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+tess = input(r"Enter tesseract path");
```
Comment (kaustubhgupta) [inline Train_pnr_status/script.py:43]:
```suggestion
tess = input(r"Enter tesseract path: ");
```

File: Train_pnr_status/Readme.md [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+    
+## Author
+   
+![Amit Kumar Mishra](https://github.com/Amit366)
```
Comment (kaustubhgupta) [inline Train_pnr_status/Readme.md:47]:
```suggestion
[Amit Kumar Mishra](https://github.com/Amit366)
```

**Review by kaustubhgupta — CHANGES_REQUESTED** [review]
- the level will be increased to 2 for GUI effort
- Can you also add the screenshot of the GUI in the readme file?
- Other things:

File: Train_pnr_status/script.py (hunk 2 above)
Comment (Amit366) [reply Train_pnr_status/script.py:42]: Sir this is to open the chrome, the path for chrome should be entered

Comment (kaustubhgupta) [reply Train_pnr_status/script.py:42]: oh then you should specify Google Chrome and not Chrome, to avoid confusions

**Comment by Amit366** [comment]
@kaustubhgupta please review

File: Train_pnr_status/Readme.md [hunk 5]
```diff
@@ -0,0 +1,44 @@
+# <b>Train PNR Status</b>
+
+[![forthebadge](https://forthebadge.com/images/badges/made-with-python.svg)](https://forthebadge.com)
+
+## Train PNR Status Functionalities : 🚀
+
+- First enter the google executable path
```
Comment (kaustubhgupta) [inline Train_pnr_status/Readme.md:7]: is this still needed?

**Review by kaustubhgupta — CHANGES_REQUESTED** [review]
- your PR is deleting GST calculator README. Kindly revert that commits
- Other change:

**Comment by Amit366** [comment]
@kaustubhgupta done.

**Comment by kaustubhgupta** [comment]
> @kaustubhgupta done.

The file is still getting deleted:
image

**Comment by Amit366** [comment]
@kaustubhgupta added

**Review by kaustubhgupta — APPROVED** [review]

**Review by antrikshmisri — APPROVED** [review]

**Review by santushtisharma10 — APPROVED** [review]

**Review by avinashkranjan — APPROVED** [review]
LGTM..🚩

**Pull request merged**